ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.12.yaml

```diff
@@ -265,6 +265,16 @@ tests:
       BASE_DOMAIN: arm-ci.devcluster.openshift.com
       COMPUTE_NODE_TYPE: m6g.xlarge
     workflow: openshift-e2e-aws-techpreview-serial
+- as: ocp-e2e-azure-ovn-arm64
+  interval: 72h
+  steps:
+    cluster_profile: azure-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: arm-ci.azure.devcluster.openshift.com
```

Comment:
need to add `OCP_ARCH: arm64` in env, default is amd64.

Reply:
Yes, I was waiting for the other PR to merge.